doc/list.rst

```diff
@@ -1092,7 +1092,7 @@ List of Available Rules
    Configuration options:
 
    - | ``syntax``
-     | Whether to use the `long` or `short` `list` syntax.
+     | Whether to use the ``long`` or ``short`` ``list`` syntax.
```

Comment:
Can we improve the source of this description and change it to:

> Whether to use the `long` or `short` syntax for array destructuring.

It does not look good with "`short` `list`".

Reply:
Yes, it looks more appropriate.